src/Apps/Artwork/Components/Seo/__tests__/SeoDataForArtwork.test.tsx

```diff
@@ -0,0 +1,253 @@
+import { MockBoot } from "DevTools"
+import { mount } from "enzyme"
+import React from "react"
+
+import { CreativeWork } from "Components/v2/Seo/CreativeWork"
+import { Product } from "Components/v2/Seo/Product"
+import { AVAILABILITY, SeoDataForArtwork } from "../SeoDataForArtwork"
+import { SeoDataForArtworkFixture } from "./SeoDataForArtwork.fixture"
+
+jest.unmock("react-relay")
+
+describe("SeoDataForArtwork", () => {
+  const getWrapper = async (artwork = SeoDataForArtworkFixture) => {
+    return mount(
+      <MockBoot>
+        <SeoDataForArtwork artwork={artwork} />
+      </MockBoot>
+    )
+  }
+
+  const getProductData = wrapper =>
+    wrapper
+      .find(Product)
+      .first()
+      .props().data
+
+  describe("SeoDataForArtworkFragmentContainer", () => {
+    it("Renders a CreativeWork for an institution", async () => {
+      const wrapper = await getWrapper({
+        ...SeoDataForArtworkFixture,
+        partner: {
+          ...SeoDataForArtworkFixture.partner,
+          type: "Institution",
+        },
+      })
+
+      expect(wrapper.find(CreativeWork).length).toEqual(1)
+
+      const data = wrapper
+        .find(CreativeWork)
+        .first()
+        .props().data
+      expect(data).toEqual({
+        brand: {
+          "@type": "Person",
+          name: "Artist McArtist",
+        },
+        description: "artwork description",
+        image: "artwork-image",
+        name: "artwork title",
+        url: "undefined/artwork/an-artwork",
```

Comment:
#minor you can mock sharify in this file and provide a `{ data: { APP_URL: "..." } }` (once you switch to using it for `APP_URL`), and then here you can test the full URL!